modules/menus/menu_cheats/src/cheat_menu.cpp

```diff
@@ -9,6 +9,13 @@
 #include "rels/include/defines.h"
 #include "menus/utils/menu_mgr.h"
 
+#ifdef GCN_PLATFORM
+#define TRANSFORM_COMBO_TEXT "R+Y"
+#endif
+#ifdef WII_PLATFORM
+#define TRANSFORM_COMBO_TEXT "C+Z"
```

Comment:
Avoid using only C+Z for Wii combos, because it is the precursor of many other combos.